meta.json

```
@@ -1,13 +1,13 @@
 {
   "$schema": "https://dl.viam.dev/module.schema.json",
-  "module_id": "hazalmestci:mcp300x-adc-sensor",
+  "module_id": "viam:mcp300x-adc-sensor",
```

I think viam-labs is the appropriate place for it right now? `viam` namespace means the module is officially supported which Im not sure it actually is.
```suggestion
  "module_id": "viam-labs:mcp300x-adc-sensor",
```